scipy/io/matlab/mio.py

```diff
@@ -207,10 +207,10 @@ def loadmat(file_name, mdict=None, appendmat=True, **kwargs):
     >>> matstruct_squeezed['teststruct'].shape
     ()
     >>> matstruct_squeezed['teststruct']['complexfield'].shape
-    ()
-    >>> matstruct_squeezed['teststruct']['complexfield'].item()
+    (3,)
+    >>> matstruct_squeezed['teststruct']['complexfield']
     array([ 1.41421356+1.41421356j,  2.71828183+2.71828183j,
-        3.14159265+3.14159265j])
```

Comment:
@smax1: I think this is proof this fixes your problem

Reply:
@eric-wieser Yes, I think so too. Thank you very much for actually making thing better. Now, I wonder how long it will take this change to get to Anaconda release...